Time.py

```diff
@@ -23,7 +23,9 @@
 OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 SOFTWARE.
 """
-"""Time.py
```

Comment:
See previous review comment about Python documentation docstrings (in `Clock.py`).